docs/data/data-grid/style/style.md

```diff
@@ -116,26 +116,64 @@ The following demo illustrates how this can be achieved.
 
 {{"demo": "StripedGrid.js", "bg": "inline"}}
 
-## Theme header and pinned sections
+## Theme container, header and pinned sections
 
-By default, the Data Grid uses the Material UI `theme.palette.background.default` color for the background of its header and pinned sections. These elements require a solid color to hide the scrollable content behind them. You can override that color with the following configuration:
+By default, the Data Grid uses the Material UI `theme.palette.background.default` color for the background color of the grid container, the column headers, and the pinned rows and columns.
+
+The various background colors can be overridden with the following configuration:
 
 ```tsx
 import { createTheme } from '@mui/material/styles';
 import type {} from '@mui/x-data-grid/themeAugmentation';
 
 const theme = createTheme({
-  mixins: {
-    MuiDataGrid: {
-      // Pinned columns sections
-      pinnedBackground: '#340606',
-      // Headers, and top & bottom fixed rows
-      containerBackground: '#343434',
+  palette: {
+    DataGrid: {
+      // Container background
+      bg: '#f8fafc',
+      // Pinned rows and columns background
+      pinnedBg: '#f1f5f9',
+      // Column header background
+      headerBg: '#eaeff5',
     },
   },
 });
 ```
 
+Material UI v6 users can use the `colorSchemes` property to specify different colors for light and dark mode:
```

Comment:
might be nice to add h3s for `### Material UI v6` and `### Material UI v5` so users can quickly differentiate between the two samples when skimming

Reply:
Makes sense! The sections are quite specific to light and dark mode theming so I've added that to the headings:

- Light and dark mode in Material UI v5
- Light and dark mode in Material UI v5